test/scripts/gen-token.mjs

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+import twilio from 'twilio';
+import { parseEnvVar, parseScriptArgument } from './common.mjs';
+
+const { AccessToken } = twilio.jwt;
+
+/**
+ * Generate a configured access token using environment variables.
+ * @param {string} identity The identity to vend the token with.
+ * @throws {Error} Will throw if any required environment variable is missing or
+ * if the identity is invalid.
+ * @returns {AccessToken}
+ */
+function generateToken(identity) {
+  if (typeof identity !== 'string') {
+    throw new Error('Identity not of type "string".');
+  }
+  if (identity === '') {
+    throw new Error('Identity evaluated to the empty string.');
+  }
+
+  const accountSid = parseEnvVar('ACCOUNT_SID');
+  const apiKeySid = parseEnvVar('API_KEY_SID');
+  const apiKeySecret = parseEnvVar('API_KEY_SECRET');
+  const outgoingApplicationSid = parseEnvVar('OUTGOING_APPLICATION_SID');
+  const pushCredentialSid = parseEnvVar('PUSH_CREDENTIAL_SID');
```

Comment:
Do we need to document these somewhere?

Reply:
Sure. Just a comment at the top of the file?

Reply:
I meant readme or something similar. For example if I'm a new contributor/team member, how do I know these requirements so I can run locally?

Reply:
I see. We don't run this script locally at the moment, maybe we file a ticket for improving local testing and documentation for local testing?

Reply:
Got it. In that case, no need to document for now.